embassy/src/time/duration.rs

```diff
@@ -61,19 +60,21 @@ impl Duration {
             .checked_add(rhs.ticks)
             .map(|ticks| Duration { ticks })
     }
+
     /// Subtracts one Duration to another, returning a new Duration or None in the event of an overflow.
     pub fn checked_sub(self, rhs: Duration) -> Option<Duration> {
         self.ticks
             .checked_sub(rhs.ticks)
             .map(|ticks| Duration { ticks })
     }
-    /// Multiplies one Duration to another, returning a new Duration or None in the event of an overflow.
 
+    /// Multiplies one Duration to another, returning a new Duration or None in the event of an overflow.
```

Comment:
This multiplies duration*u32, not duration*duration

Reply:
Oops. Looks like I moved a little too quickly there.